ghcide/src/Development/IDE/Types/Shake.hs

```diff
@@ -101,13 +101,9 @@ newtype Q k = Q (k, NormalizedFilePath)
 instance Show k => Show (Q k) where
     show (Q (k, file)) = show k ++ "; " ++ fromNormalizedFilePath file
 
--- | Invariant: the 'v' must be in normal form (fully evaluated).
```

Comment:
Why we remove this invariant🤔

Reply:
Okay, I’ll add it back. 

I think it’s accomplished by the seqValue thing or what it’s called.

Reply:
good thanks

Reply:
fixed.